### DISCUSSION TITLE
Error using prover object Src size is incorrect

### DISCUSSION THREAD

**Discussion by chianew** [Chia-Network/chia-blockchain#3373] [Q&A]
I found the log "2021-05-02T08:21:23.805 harvester chia.harvester.harvester: ERROR    Error using prover object Src size is incorrect"

Do I need to delete the ERROR  plot file?

**Comment by chianew** [comment]
I copy this plot into another computer, with chia 1.1.4, it works well
but always this log in debug file in my first PC

what's wrong?

**Reply by GeorgePapageorgakis** [reply]
What is going on with this error? I see it happening too...

**Comment by grafbcn** [comment]
I have same error. I copy file to other HDD and it don't change anything, I steal have same error "Error using prover object Src size is incorrect"

**Reply by chianew** [reply]
I deleted the incorrect plot. maybe it's the bad one ....
